.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "Project"]
+	path = Project
+	url = https://github.com/o3de/RobotVacuumSample.git
+        branch = development
```

Comment:
Use tag or hash, the development branch might be too dangerous in the future.

Reply:
The release was already made and this PR only fixes the lack of a `main` branch. I believe the issues should be solved in the `development` branch and applied to a next release.

Reply:
This is referenced in #9